test/all_parameter_combs_test.py

```diff
@@ -8,38 +8,38 @@
 from .all_modules import *  # noqa: F403
 
 parameterized_envs = [
-    ["atari/boxing_v2", boxing_v2, dict(obs_type="grayscale_image")],
-    ["atari/boxing_v2", boxing_v2, dict(obs_type="ram")],
-    ["atari/boxing_v2", boxing_v2, dict(full_action_space=False)],
-    ["atari/combat_plane_v2", combat_plane_v2, dict(game_version="jet")],
-    ["atari/combat_plane_v2", combat_plane_v2, dict(guided_missile=True)],
-    ["atari/combat_tank_v2", combat_tank_v2, dict(has_maze=True)],
-    ["atari/combat_tank_v2", combat_tank_v2, dict(is_invisible=True)],
-    ["atari/combat_tank_v2", combat_tank_v2, dict(billiard_hit=True)],
-    ["atari/maze_craze_v3", maze_craze_v3, dict(game_version="race")],
-    ["atari/maze_craze_v3", maze_craze_v3, dict(game_version="capture")],
-    ["atari/maze_craze_v3", maze_craze_v3, dict(visibilty_level=1)],
-    ["atari/maze_craze_v3", maze_craze_v3, dict(visibilty_level=3)],
-    [
-        "atari/space_invaders_v2",
-        space_invaders_v2,
-        dict(
-            alternating_control=True,
-            moving_shields=True,
-            zigzaging_bombs=True,
-            fast_bomb=True,
-            invisible_invaders=True,
-        ),
-    ],
-    ["classic/leduc_holdem_v4", leduc_holdem_v4, dict(num_players=2)],
-    ["classic/leduc_holdem_v4", leduc_holdem_v4, dict(num_players=3)],
-    ["classic/leduc_holdem_v4", leduc_holdem_v4, dict(num_players=4)],
-    ["classic/texas_holdem_v4", texas_holdem_v4, dict(num_players=2)],
-    ["classic/texas_holdem_v4", texas_holdem_v4, dict(num_players=3)],
-    ["classic/texas_holdem_v4", texas_holdem_v4, dict(num_players=4)],
-    ["classic/texas_holdem_no_limit_v6", texas_holdem_no_limit_v6, dict(num_players=2)],
-    ["classic/texas_holdem_no_limit_v6", texas_holdem_no_limit_v6, dict(num_players=3)],
-    ["classic/texas_holdem_no_limit_v6", texas_holdem_no_limit_v6, dict(num_players=4)],
+    # ["atari/boxing_v2", boxing_v2, dict(obs_type="grayscale_image")],
```

Comment:
Why are these commented out?

Reply:
Thanks for giving this a second look through. This one was just a temporary change to only test the things that were failing when I was debugging things to shorten the iteration time, will remove them and go over things again to make sure there’s no more relative imports or super calls with ezpickle

Reply:
Could you revert this change

Reply:
Oops lol sorry about that will do